code/script.js

```diff
@@ -1,9 +1,24 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
-
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+let accordionHeader = document.getElementById("accordionHeader");
+let accordionContent = document.getElementById("accordionContent");
+
+let accordionHeaderTwo = document.getElementById("accordionHeaderTwo");
+let accordionContentTwo = document.getElementById("accordionContentTwo");
+
+let accordionHeaderThree = document.getElementById("accordionHeaderThree")
+let accordionContentThree = document.getElementById("accordionContentThree")
```

Comment:
These can be declared with `const` becasue the values are not changing. Also, be consistent in whether you're using semicolons or not